iree_compiler_plugin_group.cmake

```diff
@@ -5,4 +5,7 @@
 # SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
 
 set(IREE_AMD_AIE_SOURCE_DIR "${CMAKE_CURRENT_LIST_DIR}")
-add_subdirectory(${CMAKE_CURRENT_LIST_DIR}/compiler/plugins/target/AMD-AIE target/AMD-AIE)
\ No newline at end of file
+add_subdirectory(${CMAKE_CURRENT_LIST_DIR}/compiler/plugins/target/AMD-AIE target/AMD-AIE)
+if(ADD_XRT_RUNTIME)
+    add_subdirectory(${CMAKE_CURRENT_LIST_DIR}/runtime/plugins/XRT XRT)
```

Comment:
Ideally we aren't bundling this into the compiler plugin system but I know why you are (it is easy/accessible).

I need to do a bit more cmake hacking to make this separable in a similar way that the compiler side is (basically, we should be able to build compiler and runtime separately).

Ultimately, we will want to use the `IREE_EXTERNAL_TOOLING_MODULES` facility (e.g. https://github.com/openxla/openxla-nvgpu/blob/main/CMakeLists.txt#L93) to provide additional features. This plumbing has been used a few times but it may need a bit more work. I can look at that after you land.

Reply:
we will be working on improving this in the following PRs